cfme/tests/infrastructure/test_timelines.py

```diff
@@ -103,99 +152,214 @@ def _power_on(self):
     def _power_off(self):
         return self.vm.provider.mgmt.stop_vm(self.vm.name)
 
+    def _restart_vm(self):
+        return self._power_off() and self._power_on()
+
     def _suspend(self):
-        return self.vm.provider.mgmt.suspend_vm(self.vm.name)
+        return (self.vm.provider.mgmt.suspend_vm(self.vm.name) and
+                self.vm.provider.mgmt.start_vm(self.vm.name))
 
     def _rename_vm(self):
         logger.info('%r will be renamed', self.vm.name)
         new_name = self.vm.provider.mgmt.rename_vm(self.vm.name, self.vm.name + '-renamed')
         logger.info('%r new name is %r', self.vm.name, new_name)
         self.vm.name = new_name
+        logger.info('self.vm.name is now %r', self.vm.name)
+        logger.info('%r will be rebooted', self.vm.name)
         self.vm.provider.mgmt.restart_vm(self.vm.name)
         return self.vm.name
 
-    def _check_timelines(self, target):
-        """Navigate to the TL of the given target, select the category of the event and verify
-        that the tl_event of the VMEvent is present. if will return the length of the array
-        containing  the events found in that timeline.
+    def _delete_vm(self):
+        logger.info('%r will be deleted.', self.vm.name)
+        return self.vm.provider.mgmt.delete_vm(self.vm.name)
 
+    def _clone_vm(self):
+        logger.info('%r will be cloned to %r.', self.vm.name, self.vm.name + '-clone')
+        clone_name = self.vm.name + '-clone'
```

Comment:
REQUIRED: please move it about logger statement and re-use it in logger call.

Reply:
ok done